vermouth/dssp/dssp.py

```diff
@@ -156,26 +224,24 @@ def run_dssp(system, executable='dssp', savefile=None, defer_writing=True):
     In order to call DSSP, a PDB file is produced. Therefore, all the molecules
     in the system must contain the required attributes for such a file to be
     generated. Also, the atom names are assumed to be compatible with the
-    'universal' force field for DSSP to recognize them.
+    'charmm' force field for DSSP to recognize them.
```

Comment:
Just the name change. DSSP *is* picky about atom names, but I don't know to what extent (see above).
Here I just changed universal to charmm, since the ff universal doesn't exist anymore